libs/utils/src/generation.rs

```diff
@@ -89,6 +89,22 @@ impl Generation {
             Self::Broken => panic!("Attempted to use a broken generation"),
         }
     }
+
+    pub fn next(&self) -> Generation {
```

Comment:
I originally wrote that, but it didn't work, probably because the test using the method is in a different crate, so this code isn't getting compiled with `test` when testing.